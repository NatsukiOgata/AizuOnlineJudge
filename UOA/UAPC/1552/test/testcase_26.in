1000 6 2 12 3 8 7 1 14 13 15 10 11 3 4 4 10 16 7 17 14 14 6 2 19 17 8 17 17 6 10 6 13 18 19 14 6 17 19 6 18 10 19 19 16 13 3 0 11 14 18 10 13 12 4 12 19 2 13 12 7 15 0 17 16 9 2 9 11 5 9 15 18 11 16 1 17 18 3 16 16 17 10 12 4 14 8 0 2 7 17 11 18 18 1 11 6 10 9 9 2 16 9 12 9 3 10 3 17 16 2 1 1 7 19 11 5 4 2 2 16 2 17 16 2 11 0 17 11 15 9 13 18 11 17 6 1 0 4 16 6 4 9 18 10 6 1 11 4 7 7 15 4 18 14 9 17 7 7 2 15 8 10 9 13 7 18 8 7 8 15 19 17 9 5 8 6 4 3 19 9 2 7 7 10 19 19 11 12 13 10 9 6 13 1 6 17 4 19 19 11 0 1 4 6 2 15 7 4 0 7 0 2 12 18 3 14 3 0 17 16 1 3 4 18 7 16 8 7 15 12 5 11 14 16 13 12 11 1 18 14 5 13 12 13 7 11 13 4 10 2 2 18 2 17 9 9 6 8 7 11 6 16 4 4 9 18 0 14 0 8 15 8 8 5 9 4 5 13 11 18 19 10 1 5 10 13 18 3 0 17 16 15 9 13 8 3 7 15 16 19 17 13 10 14 3 18 14 11 8 12 12 17 11 4 8 4 18 16 11 3 0 2 8 13 4 13 17 15 8 12 19 17 6 13 3 11 7 8 8 13 16 9 2 15 15 19 7 9 10 17 14 0 4 17 11 17 8 4 0 19 2 12 9 7 10 1 19 9 0 8 15 6 14 15 4 19 0 17 10 3 5 11 5 16 4 7 10 6 16 5 1 3 12 11 1 19 9 6 14 0 8 13 19 15 17 2 8 0 5 7 6 7 16 3 16 10 6 11 3 9 0 17 6 6 19 12 8 15 1 11 10 17 11 9 10 15 6 11 14 19 13 19 7 12 14 6 5 19 6 19 17 4 16 19 5 13 15 1 16 6 14 19 0 12 5 2 14 19 10 6 8 8 10 8 1 19 19 3 4 8 13 0 16 12 5 17 13 6 10 19 0 14 16 13 2 11 15 8 11 5 6 16 9 8 2 7 10 4 16 7 4 6 16 18 19 0 15 10 2 4 5 8 9 18 1 14 17 7 4 4 6 2 11 5 0 4 16 8 17 18 4 11 17 11 3 4 19 5 17 15 14 14 15 16 1 3 10 4 10 9 2 1 0 16 1 19 19 14 9 5 8 6 7 14 17 3 4 4 0 17 10 1 8 9 15 15 2 12 10 0 2 13 7 19 10 13 2 9 14 16 3 7 9 8 9 5 16 18 6 12 7 11 13 10 13 15 16 3 4 10 10 16 10 18 15 10 16 9 6 8 9 13 1 14 19 9 1 4 2 5 15 0 17 18 3 3 19 7 16 0 19 0 12 18 16 6 16 7 9 7 7 11 15 13 15 0 10 7 4 18 3 3 3 13 12 10 11 18 17 18 1 17 0 12 18 10 8 6 18 18 12 14 6 2 19 13 5 19 12 6 17 1 16 3 2 5 14 12 8 3 18 15 13 16 15 2 15 16 14 9 13 2 15 5 15 12 10 10 1 0 0 15 2 2 7 19 17 6 16 1 7 11 19 0 8 18 0 11 7 11 14 8 17 19 19 1 16 17 2 18 2 4 19 16 11 5 5 8 0 12 1 15 1 0 5 15 16 10 18 2 6 14 6 2 11 1 7 5 6 9 10 1 6 7 8 9 8 1 4 12 16 10 2 1 3 7 11 13 10 11 7 13 14 8 16 14 18 9 6 15 0 7 19 6 10 4 1 2 8 12 10 10 2 0 7 18 3 5 9 18 18 0 12 4 13 14 17 13 16 15 3 0 14 10 19 3 7 5 17 9 18 14 19 11 19 3 8 11 19 11 17 12 10 7 7 5 4 19 10 17 17 18 4 6 12 6 2 15 11 10 14 2 18 9 13 12 12 14 1 9 4 7 10 7 8 14 0 19 12 3 11 1 10 13 8 6 13 18 17 7 17 2 1 19 15 12 11 6 17 17 9 4 13 14 19 15 1 19 17 11 6 14 11 1 11 0 15 16 14 1 11 7 2 17 18 18 10 17 3 5 18 11 4 2 4 9 6 14 5 1 17 13 19 3 17 0 10 9 2 16 5 17 13 17 17 15 14 0 2 4 11 0 2 9 0 6 4 3 6 17
