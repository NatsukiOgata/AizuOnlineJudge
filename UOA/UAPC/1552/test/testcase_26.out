15 14 18 295 297
